frontend/wailsjs/go/backend/App.js

```diff
@@ -5,3 +5,7 @@
 export function StartBuild() {
   return window['go']['backend']['App']['StartBuild']();
 }
+
+export function StartGitHubLogin() {
+  return window['go']['backend']['App']['StartGitHubLogin']();
+}
```

Comment:
so the way wails works (just reading this) is it builds a js file and sticks it on the `window.` for the page that these bindings then call. this notation works just like dot notation like this could be `window.go.backend.App.etc` but I assume they used this because dot notation doesn't work if like the name of the thing has a period in it. so this is a wa yto work around that. 

the bindings must rpc or restfully call the backend. I haven't looked at exactly how that works yet